source/features/feature-descriptions.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCleanPathname, getCurrentBranch, getRepositoryInfo, upperCaseFirst} from '../github-helpers';
+
+function toggleHandler({delegateTarget: button}: delegate.Event): void {
+	const isHidden = select('.rgh-feature-screenshot')!.classList.toggle('d-none');
+	if (isHidden) {
+		button.textContent = 'View Screenshot';
+	} else {
+		button.textContent = 'Hide Screenshot';
+	}
+}
+
+async function init(): Promise<void | false> {
+	const currentFeature = location.pathname.split('/').pop()!.replace(/.tsx|.css/, '');
```

Comment:
Can you find a single regex that either replaces everything but the name, without needing the extension? If that's too hard, `.match` could instead be used to extract it.